From e17e632a0a3cb59c0db7a08ddf2e1cb7bc4b6ca1 Mon Sep 17 00:00:00 2001
From: Michael Anthony Knyszek <mknyszek@google.com>
Date: Fri, 11 Nov 2022 17:26:45 +0000
Subject: [PATCH 22/32] net: disable TestLookupDotsWithRemoteSource and
 TestLookupGoogleSRV

These tests fail consistently due to a DNS change causing widespread
trybot outages.

Fixes #56707.

Reviewed-on: https://go-review.googlesource.com/c/go/+/449640
Reviewed-by: Carlos Amedee <carlos@golang.org>
Reviewed-by: Bryan Mills <bcmills@google.com>
Run-TryBot: Michael Knyszek <mknyszek@google.com>
TryBot-Result: Gopher Robot <gobot@golang.org>
Reviewed-by: Damien Neil <dneil@google.com>
Change-Id: I7ea7ed1f701ce5a5f930885a9c817fdebd6aee4d
---
 src/net/lookup_test.go | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/net/lookup_test.go b/src/net/lookup_test.go
index fa1a706c78..cb2b64f0b5 100644
--- a/src/net/lookup_test.go
+++ b/src/net/lookup_test.go
@@ -71,6 +71,10 @@ var lookupGoogleSRVTests = []struct {
 var backoffDuration = [...]time.Duration{time.Second, 5 * time.Second, 30 * time.Second}
 
 func TestLookupGoogleSRV(t *testing.T) {
+	// TODO(mknyszek): Figure out next steps for this test. This is just
+	// a quick fix.
+	t.Skip("fails consistently due to an upstream DNS change; see #56707.")
+
 	t.Parallel()
 	mustHaveExternalNetwork(t)
 
@@ -633,6 +637,10 @@ func TestLookupDotsWithLocalSource(t *testing.T) {
 }
 
 func TestLookupDotsWithRemoteSource(t *testing.T) {
+	// TODO(mknyszek): Figure out next steps for this test. This is just
+	// a quick fix.
+	t.Skip("fails consistently due to an upstream DNS change; see #56707.")
+
 	if runtime.GOOS == "darwin" || runtime.GOOS == "ios" {
 		testenv.SkipFlaky(t, 27992)
 	}
-- 
2.40.0

